README.md

```diff
@@ -1 +1,58 @@
-# HTML-CSS-JavaScript-capstone-project
\ No newline at end of file
+ Description of the project: 
+    This project represents the mobile version and the Desktop version of the first capstone project from Microverse, Where I used all the skills and knowledge that I acquired form the first module.
+
+
+## Built With
+
+- Major languages: HTML
+-  CSS
+    - Flexbox
+    - CSS Grid
+- JavaScript
+- Mobile first aproach
+- linter for code analysis
+
+## Deployment
+https://mohammedelseddik.github.io/Web-Development-Capstone-Project/
```

Comment:
This link leads me to a 404 Page not found.

- [ ] Please make sure that your deployment works as intended.
![image](https://user-images.githubusercontent.com/83596135/157761877-1ba2a2bf-bbd5-4061-affb-f703d4d290fb.png)